### ISSUE TITLE
Posting message with web interface would be better to be disabled by default

### ISSUE THREAD

**Issue by ikedas** [sympa-community/sympa#1473]
Posting message using web interface let the mailing list server send messages authored by the user on the other mail domain.

When it is allowed, message delivery to the recipients (subscribers) with domain authentication will fail. This could not only cause those subscribers to be treated as bouncers, but could also degrade the reputation of the mailing list service.

Expected Behavior
-----------------

Posting message using web interface would be better to be disabled by default, and may be enabled in the safe case, e.g. messages are anoymized, DMARC protection is enabled.

Current Behavior
----------------

Posting message using web interface is enabled, unless `send` scenario prevents it.

Context
-------

Currently the list server of `sympa.community` supports DKIM signing and ARC sealing, and the messages are delivered to subscribers on the domain forcing DMARC policy.  However, when some users posted via web interface, it was reported that delivery to some subscribers refused.  After consideration, we have concluded that our Sympa server cannot provide posting via its web interface.

**Comment by racke** [comment]
We are using the web interface for HTML newsletters. DKIM signing is done by the MTA. So it would be good to have a flag to enable posting from the web interface in the future.

**Comment by ikedas** [comment]
I agree.  Listmasters who know what they do should feel free to enable posting manually.

**Comment by Neustradamus** [comment]
@ikedas: Have you progressed to fix the web interface?

Linked to:
- https://github.com/sympa-community/sympa/issues/1814
